adapters/adapterstest/test_json.go

```diff
@@ -22,38 +22,37 @@ import (
 // RunJSONBidderTest is a helper method intended to unit test Bidders' adapters.
 // It requires that:
 //
-//   1. Bidders communicate with external servers over HTTP.
-//   2. The HTTP request bodies are legal JSON.
+//  1. Bidders communicate with external servers over HTTP.
+//  2. The HTTP request bodies are legal JSON.
```

Comment:
To other reviewers, the following comment changes are autogenerated. Go 1.19 auto formats comments while earlier versions of Go do not.